src/mame/commodore/kim1.cpp

```diff
@@ -226,6 +259,9 @@ void kim1_state::u2_write_b(uint8_t data)
 	// Cassette write/speaker update
 	if (data & 0x20)
 		m_cass->output((data & 0x80) ? -1.0 : 1.0);
+
+	// Write bit 0 to serial console
+	m_rs232->write_txd(data & 0x01);
```

Comment:
You need to keep the state of line in `tty_callback` and AND it with bit 0 here to avoid issues when they’re both changing at once.